webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -53,6 +53,15 @@ const ActionsBar = ({
   const deleteHostHandler = () =>
     dispatch(deleteHost(hostId, computeId, destroyVmOnHostDelete));
 
+  const determineToolTip = () => {
+    if (!(computeId && isHostActive)) {
+      if (computeId) {
+        return __('Console disabled as the host is powered off.');
+      }
+      return __('Compute resource does not support the console function.');
+    }
+    return undefined;
```

Comment:
nit: can be simplified into `return;`

Reply:
This doesn't pass the lint test. And you can't remove the `return` either because arrow function expects a return value.